internal/xds/rbac/converter_test.go

```diff
@@ -169,3 +167,18 @@ func createStdoutPb(t *testing.T) *anypb.Any {
 	}
 	return customConfig
 }
+
+// Builds a config with a nonsensical type in the anypb.Any.
+func createUnsupportedPb(t *testing.T) *anypb.Any {
```

Comment:
I think you might be able to re-use https://github.com/grpc/grpc-go/blob/59134c303c31ba2ce65da8962c050918098f2a25/internal/testutils/marshal_any.go#L30 instead to simplify a little?

Reply:
Changed